src/client/client.ts

```diff
@@ -61,7 +62,7 @@ export type PublishRequest<TBody = BodyInit> = {
    *
    * @default undefined
    */
-  delay?: number;
+  delay?: Duration | number;
```

Comment:
should update docstring too with Duration

Reply:
Yeah, after the merge I'll do it.